libraries/nginx_service.rb

```diff
@@ -94,6 +96,7 @@ def action_enable
       directory '/run/nginx' do
```

Comment:
I would ask @acaiafa about that.
@johnbellone , what I found is that nginx needed to run as root for the service to actually come up on rhel.
I am assuming we dont want to do that. But I couldn't find an easy alternative.